### PR TITLE
Add fpx website

### PR DESCRIPTION
**Pull request by laulauland**
This is a WIP PR for the upcoming FPX landing/docs page.

It uses Astro (Starlight) for the docs page, Tailwind for styling (to align with `fpx` itself), and MDX for the content.

### CODE REVIEW

File: www/src/content/docs/home/get-started.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { instrument } from '@fiberplane/hono-otel';
+
+const app = new Hono<{ Bindings: Bindings }>()
+
+
+app.get('/', (c) => {
+  return c.text('Hello Hono!')
+})
+
+export default app
```
Comment (mies) [inline www/src/content/docs/home/get-started.md]: export default instrument(app)

File: www/src/content/docs/home/get-started.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ npm create hono@latest my-app 
+ ```
+
+
+If you want a project including an ORM, a database and already has a cloud deployment platform in mind we recommend using [HONC](https://honc.dev/):
+
+```
+create honc-app@latest 
+```
+
```
Comment (mies) [inline www/src/content/docs/home/get-started.md:21]: also need to install middleware:

npm i@fiberplane/hono-otel

((maybe needs specific version/tag))

Comment (Nlea) [reply www/src/content/docs/home/get-started.md:21]: updated in the docs. We haven't published the npm package yet right?

Comment (laulauland) [reply www/src/content/docs/home/get-started.md:21]: it's up! Alpha version and no proper readme yet, but it's a prio for this week for us: https://www.npmjs.com/package/@fiberplane/hono-otel

**Review by mies — APPROVED** [review]

**Comment by brettimus** [comment]
my understanding is we're going to merge this as a landing page only build, then create a new branch to expand content, so i've marked as ready for review (pending updated screenshots)

**Pull request merged**